starter-code/src/functions-and-arrays.js

```diff
@@ -10,16 +14,39 @@ var words = [
   'orchard',
   'crackpot'
 ];
+function findLongestWord(myArray){
+  if(myArray.length == 0) return undefined;
```

Comment:
you're doing a lot of unneeded check here- your code works without these two lines. You just need to change `>`to be `>=` 

```
function findLongestWord(myArray) {
  if (myArray.length == 0) return undefined;

  return myArray.reduce((acc, currVal) => {
    return (acc.length >= currVal.length) ? acc : currVal
  }, 0
  )
} 
```